packages/admin/resources/views/components/layouts/app.blade.php

```diff
@@ -18,9 +18,9 @@ class="fixed inset-0 z-20 w-full h-full bg-gray-900/50 lg:hidden filament-sideba
         @if (config('filament.layout.sidebar.is_collapsible_on_desktop'))
             <div
                 x-data="{}"
-                class="hidden transition-all w-screen space-y-6 flex-1 flex flex-col filament-main lg:pl-80 rtl:lg:pl-0 rtl:lg:pr-80"
```

Comment:
The `hidden` class needs to remain on this line, it's not ambiguous as it's then being overridden by `!flex` to act like `x-cloak`

Reply:
Done